db/views/reporting_prescriptions_v01.sql

```diff
@@ -0,0 +1,43 @@
+SELECT p.id as patient_id,
+       p.month_date,
+       COALESCE(SUM(clean_dosage) FILTER (WHERE clean_name = 'Amlodipine'), 0)          as amlodipine,
+       COALESCE(SUM(clean_dosage) FILTER (WHERE clean_name = 'Telmisartan'), 0)         as telmisartan,
+       COALESCE(SUM(clean_dosage) FILTER (WHERE clean_name = 'Losartan'), 0)            as losartan,
+       COALESCE(SUM(clean_dosage) FILTER (WHERE clean_name = 'Atenolol'), 0)            as atenolol,
+       COALESCE(SUM(clean_dosage) FILTER (WHERE clean_name = 'Enalapril'), 0)           as enalapril,
+       COALESCE(SUM(clean_dosage) FILTER (WHERE clean_name = 'Chlorthalidone'), 0)      as chlorthalidone,
+       COALESCE(SUM(clean_dosage) FILTER (WHERE clean_name = 'Hydrochlorothiazide'), 0) as hydrochlorothiazide,
+       COALESCE(SUM(clean_dosage)
+           FILTER (
+              WHERE clean_name NOT IN ('Amlodipine', 'Telmisartan', 'Losartan', 'Atenolol', 'Enalapril', 'Chlorthalidone', 'Hydrochlorothiazide')
+              AND medicine_purpose_hypertension = true),
+           0) as other_bp_medications
+FROM (
+         SELECT *
+         FROM patients p
+         LEFT OUTER JOIN reporting_months cal
+         ON to_char(p.recorded_at AT TIME ZONE 'utc' AT TIME ZONE (SELECT current_setting('TIMEZONE')), 'YYYY-MM') <= cal.month_string
+     ) p
+LEFT JOIN LATERAL (
+    SELECT actual.name          as actual_name,
+           actual.dosage        as actual_dosage,
+           clean.medicine       as clean_name,
+           clean.dosage         as clean_dosage,
+           purpose.hypertension as medicine_purpose_hypertension,
+           purpose.diabetes     as medicine_purpose_diabetes
+    FROM prescription_drugs actual
+             LEFT JOIN raw_to_clean_medicine raw
+                       ON lower(regexp_replace(raw.raw_name, '\s+', '', 'g')) =
+                          lower(regexp_replace(actual.name, '\s+', '', 'g'))
```

Comment:
Could add a regex_replaced/slugified version in the tables while importing data to simplify the query.

Reply:
Right now the import is a plain `COPY`. And actually, since these regex/lower will remain in the query for the actual prescription drugs rows, it might be better to keep it in the same place for readability and maintainability.

Reply:
Moved to a csv import with https://github.com/simpledotorg/simple-server/pull/3085. But as I said earlier, it makes sense to keep the `regexp_replace` in both LHS and RHS of the query for maintainability.